decred/decred/dcr/vsp.py

```diff
@@ -30,12 +30,7 @@ def resultIsSuccess(res):
     Returns:
         bool: True if result fields indicate success.
     """
-    return (
-        res
-        and isinstance(res, object)
-        and "status" in res
-        and res["status"] == "success"
-    )
+    return isinstance(res, object) and "status" in res and res["status"] == "success"
```

Comment:
I haven't figured it out yet, but for some reason this returns an empty dictionary if you leave in res and pass an empty dictionary.